frontend/app/tests/unit/specs/data/subscript-number.spec.ts

```diff
@@ -0,0 +1,196 @@
+import { mount } from '@vue/test-utils';
+import { createPinia, setActivePinia } from 'pinia';
+import { beforeEach, describe, expect, it } from 'vitest';
+import { BigNumber, bigNumberify } from '@rotki/common';
+import AmountDisplay from '@/components/display/amount/AmountDisplay.vue';
+import { useFrontendSettingsStore } from '@/store/settings/frontend';
+import { useGeneralSettingsStore } from '@/store/settings/general';
+
+describe('amountDisplay subscript', () => {
```

Comment:
Shouldn't these tests be part of amount-display.spec.ts? you can do an extra `describe` for subscript related test cases.